armi/materials/inconelX750.py

```diff
@@ -62,9 +63,9 @@ def setDefaultMassFracs(self):
 
     def polyfitThermalConductivity(self, power=2):
         r"""
-        Calculates the coefficients of a polynomial fit for thermalConductivity.
-        Based on data from http://www.specialmetals.com/documents/Inconel%20alloy%20X-750.pdf
-        Fits a polynomial to the data set and returns the coefficients.
+        Calculates the coefficients of a polynomial fit for thermalConductivity. Based on data from
+        http://www.specialmetals.com/documents/Inconel%20alloy%20X-750.pdf Fits a polynomial to the
```

Comment:
This link returns 
![image](https://github.com/terrapower/armi/assets/18561046/4d5ed358-eb39-4894-b91a-93f0cc865278)

Reply:
Of course it does.  Let me see if the archives have it.

Reply:
Fixed it.  Archive.org has a permanent link to that PDF:

https://web.archive.org/web/20170215105917/http://www.specialmetals.com:80/documents/Inconel%20alloy%20X-750.pdf